webapp/app/partials/perf/comparectrl.html

```diff
@@ -0,0 +1,44 @@
+<nav class="navbar navbar-default" role="navigation">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <a class="navbar-brand" href="#">Perfherder Compare Talos</a>
+    </div>
+  </div>
+</nav>
+
+<div class="ph-horizontal-layout">
+  <div id="revision-chooser">
+    <button class="btn btn-primary" ng-click="addCompareData()">
+      <span class="glyphicon glyphicon-plus" aria-hidden="true"></span> Compare Revisions</button>
+  </div>
+  <div id="data-display">
+    <div id="datapoint-detail" ng-show="originalRevision && newRevision">
+      <div id="subtest-summary">
+        <h4>Compare All Data for Revision {{newProject}} : {{newRevision}} to {{originalProject}} : {{originalRevision}}</h4>
+        <p class="help-block">Hover over each entry for more options.</p>
+        <table class="table">
+          <thead>
+            <tr>
+              <th></th><th>Test/Platform</th><th>Original</th><th>Variation</th><th>New</th><th>Variation</th><th>Delta</th><th>Delta %</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr ng-class="{'subtestheader': compareResult.isHeader}" ng-repeat="compareResult in compareResults">
+              <td><a ng-style="compareResult.isHeader && {'display': 'none'}" ng-href="{{compareResult.detailsLink}}">Details</a></td>
+              <td>{{compareResult.name}}</td>
+              <td ng-attr-title="runs: {{compareResult.originalRuns}}">{{compareResult.originalGeoMean}}</td><td>{{compareResult.originalVariation}}</td>
+              <td ng-attr-title="runs: {{compareResult.newRuns}}">{{compareResult.newGeoMean}}</td><td>{{compareResult.newVariation}}</td>
+              <td ng-style="compareResult.type === 'regression' && {'background-color': 'red', 'color': 'white', 'font-weight': 'bold'} ||
```

Comment:
We should define these as classes in perf.css, instead of setting raw CSS values conditionally in the html. That makes it easier for people who are of more of a web designer bent to contribute.